### ISSUE TITLE
helm plugin install gitrepo not working anymore with Kubernetes v1.16 

### ISSUE THREAD

**Issue by paulozagaloneves** [chartmuseum/helm-push#55]
**Kubernetes:** Microk8s v1.16

When installing helm-push plugin in kubernetes v1.16 getting the error:

```
$ helm plugin install https://github.com/chartmuseum/helm-push
Error: Unable to get repository: Cloning into '/home/pauloneves/.helm/cache/plugins/https-github.com-chartmuseum-helm-push'...
/usr/lib/git-core/git-remote-https: relocation error: /usr/lib/git-core/git-remote-https: symbol curl_global_sslset version CURL_GNUTLS_3 not defined in file libcurl-gnutls.so.4 with link time reference
: exit status 128
```
**workaround:**
```
$ git clone https://github.com/chartmuseum/helm-push https-github.com-chartmuseum-helm-push
$ helm plugin install https-github.com-chartmuseum-helm-push
```

**Comment by LukasGentele** [comment]
I had the same issue. It works for me when I add `.git`: `helm plugin install https://github.com/chartmuseum/helm-push.git`

Platform: Windows 10

**Comment by jdolitsky** [comment]
fixed in #58
